static/app/views/dashboardsV2/widgetBuilder/footer.tsx

```diff
@@ -29,7 +36,12 @@ export function Footer({goBackLocation, onSave, onDelete, isEditing}: Props) {
             <Button priority="danger">{t('Delete')}</Button>
           </Confirm>
         )}
-        <Button priority="primary" onClick={onSave}>
+        <Button
+          priority="primary"
+          onClick={onSave}
+          disabled={invalidForm}
+          title={invalidForm ? t('Required fields must be filled out') : undefined}
```

Comment:
Is this the only possible invalid state?

Reply:
for now yes...the other ones usually have some value by default , but we will double check it